phpBB/phpbb/message/message.php

```diff
@@ -258,15 +263,19 @@ public function send(\phpbb\di\service_collection $messenger, $contact)
 					$messenger_method->set_addresses($recipient);
 					$messenger_method->reply_to($this->sender_address);
 
-					$messenger_method->header('X-AntiAbuse', 'Board servername - ' . $this->server_name);
-					$messenger_method->header('X-AntiAbuse', 'User IP - ' . $this->sender_ip);
-					if ($this->sender_id)
-					{
-						$messenger_method->header('X-AntiAbuse', 'User_id - ' . $this->sender_id);
-					}
-					if ($this->sender_username)
+					if (isset($messenger_method->headers) && $messenger_method->headers instanceof \Symfony\Component\Mime\Header\Headers)
```

Comment:
With the header() function now doing nothing for other methods I don't think you need to check headers? Also, headers shouldn't be accessible here.